techniques/aria/ARIA12.html

```diff
@@ -1,29 +1,37 @@
+			<p class="type">Type: Technique</p>
+		</section>
+		<section id="applicability">
+			<h2>When to Use</h2>
       <p>Technologies that support <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>. </p>
-   </section><section id="description"><h2>Description</h2>
+   </section>
+	 <section id="description">
+		<h2>Description</h2>
       <p>The purpose of this technique is to provide a way for Assistive Technologies (AT) to identify 
-a piece of content as a heading. Applying <code class="language-html">role="heading"</code> to an element causes an AT (like a screen reader) to treat it as though it were a heading. 
-</p>
-      <p>If there is more than one heading on the page and the heading hierarchy is defined through the 
-visual presentation, the <code class="language-html">aria-level</code> attribute should be used to indicate the hierarchical level of the heading. </p>
-      <p>When possible, use native heading mark-up directly. For example, it is preferable 
-to use <code class="language-html">h1</code> rather than using <code class="language-html">&lt;div role="heading" aria-level="1"&gt;</code>. However, the use of the 
-heading role, instead of heading mark-up, may be necessary. For example, when retrofitting a legacy site where scripts depend on the existing element hierarchy.
-</p>
-      <p>The use of the <code class="language-html">heading</code> role and nesting levels is discussed in <a href="https://www.w3.org/WAI/ARIA/apg/">WAI-ARIA Authoring Practices Guide</a>.
-</p>
-   </section><section id="examples"><h2>Examples</h2>
-      <section class="example">
-         <h3>Simple headings</h3>
-         
-            <p>This example demonstrates how to implement simple headings using <code class="language-html">role="heading"</code> when retrofitting a legacy site where scripts depend on the existing element hierarchy or the level is unknown. For example, web content which is syndicated from various sources may be constructed without knowledge of what the final presentation will be.
-</p>
-<pre xml:space="preserve"><code class="language-html">&lt;div role="heading"&gt;Global News items&lt;/div&gt;
+a piece of content as a heading. Applying <code class="language-html">role="heading"</code> to an element causes an AT (like a screen reader) to treat it as though it were a heading. The <code class="language-html">role="heading"</code> property must be paired with the <code class="language-html">aria-level</code> property to define the heading level.</p>
+      <p>When possible, <a href="https://www.w3.org/TR/using-aria/#rule1">use native heading mark-up</a>. For example, it is preferable to use <code class="language-html">h1</code> rather than using <code class="language-html">&lt;div role="heading" aria-level="1"&gt;</code>. However, the use of the heading role, instead of heading <abbr title="HyperText Markup Language">HTML</abbr>, may be necessary.</p>
```

Comment:
change 'mark-up' to match the 'markup' used in the `title` for the `<abbr>HTML</abbr>`.

I realize the last sentence of this was a bit of a rejiggering of what the previous doc was saying. But it struck me a bit odd to end this saying to use the heading role when necessary, but not actually outline when or why it could be necessary.  The fact that the APG content no longer exists emphasizes the ambiguity of when it might be necessary.

So, rather than that, i wrote up a bit to explain why / double down on why it's better to use the native HTML element instead.  If people think this is "too much", then we do not need to merge in this suggestion.  I'd just rather see there be a bit more explanation, again, particularly due to the fact the APG resource is no longer available - so it may be harder for some to determine when it's necessary, or not. 

```suggestion
      <p>When possible, <a href="https://www.w3.org/TR/using-aria/#rule1">use native heading markup</a>. For example, it is preferable to use an <code class="language-html">h1</code> element, rather than using <code class="language-html">&lt;div role="heading" aria-level="1"&gt;</code>. Native <abbr title="HyperText Markup Language">HTML</abbr> elements have implicit styling and features that won't necessarily be replicated with ARIA equivalents. For instance, heading elements will have specific styling adjustments made when viewing a page in a browser's Reader Mode. Whereas a <code class="language-html">&lt;div role="heading" aria-level="1"&gt;</code> will not be styled as a heading, because the underlying element is a <code class="language-html">div</code>. ARIA only modifies the way an element is exposed to assistive technology, it does nothing to change the implicit style or functionality of the element it is applied to.</p>
```